apps/studio/components/interfaces/Linter/LinterDataGrid.tsx

```diff
@@ -69,7 +69,11 @@ const LinterDataGrid = ({
       value: (row: any) => (
         <div className="flex items-center gap-1 text-xs">
           <span className="shrink-0">{entityTypeIcon(row.metadata?.type)}</span>
-          {`${row.metadata.schema}.${row.metadata.name}`}
+          {row.metadata &&
```

Comment:
Do we need that many checks? What about just checking if schema and name are undefined?

Otherwise if possible lets shift this condition into a const so its a bit more readable

Reply:
Same for below L223

Reply:
Cleaned up 👍